Empire/EmpireManager.cpp

```diff
@@ -2,13 +2,25 @@
 
 #include "Empire.h"
 #include "../util/Directories.h"
+#include "../util/GameRules.h"
 #include "../util/Logger.h"
 #include "../util/XMLDoc.h"
 #include "../universe/Enums.h"
 
 #include <boost/filesystem/fstream.hpp>
 
 namespace {
+    void AddRules(GameRules& rules) {
```

Comment:
Why is this here and not in Diplomacy.cpp?

Reply:
Because it is used only here.

Reply:
Moved code to `Dimplomacy.cpp`